app/assets/stylesheets/application.scss

```diff
@@ -14,3 +14,16 @@
  @import 'bootstrap';
  @import 'rails_bootstrap_forms';
  @import 'badge';
+
+.feijoa-background {
+  background-image: linear-gradient(rgba(0,0,0,0.6), rgba(0,0,0,0.9)), image-url("feijoa.jpg");
```

Comment:
Should these be in variables instead?

Reply:
Which parts? I did write this CSS in a bit of a hurry so it could definitely be refactored. 
The linear-gradient is really just a hack to have a partially transparent layer over top of the image (so the text is readable).